great_expectations/checkpoint/configurator.py

```diff
@@ -24,21 +24,30 @@
 logger = logging.getLogger(__name__)
 
 
+class ActionDetails(TypedDict):
```

Comment:
This is very nice as a bandaid.  Ultimately, we should obviate `Configurator` by a more elegant design, based on the principle of broader `Checkpoint` API being offered based on the user's growth in sophistication.